native/src/modules/app/components/App.js

```diff
@@ -14,8 +14,22 @@ import NavigatorContainer from '../containers/NavigatorContainer'
 import { SafeAreaProvider } from 'react-native-safe-area-context'
 import StaticServerProvider from '../../static-server/containers/StaticServerProvider'
 import I18nProvider from '../../i18n/components/I18nProvider'
-import { NavigationContainer } from '@react-navigation/native'
+import { NavigationContainer, type LinkingOptions } from '@react-navigation/native'
 import PermissionSnackbarContainer from '../../layout/containers/PermissionSnackbarContainer'
+import { REDIRECT_ROUTE } from 'api-client'
+
+const linking: LinkingOptions = {
+  prefixes: ['https://', 'integreat://'],
+  // $FlowFixMe redirect is part of available routes
+  config: {
+    screens: {
+      [REDIRECT_ROUTE]: '*'
+    }
+  },
+  getStateFromPath: path => {
+    return { index: 0, routes: [{ name: REDIRECT_ROUTE, params: { url: `https://${path}` } }] }
+  }
+}
```

Comment:
I think this should be part of the NavigationContainer

Reply:
That is not possible. NavigationContainer is a component from react-navigation.